### PR TITLE
new coverage tests

### PR DESCRIPTION
**Pull request by hugo-dc**
Related to https://github.com/ewasm/tests/issues/85

Includes the following test cases:
-  `selfDestructNonExistingAcct`: selfdestruct to a non-existent account
-  `createReturnEmptyData`: call/create return empty data
-  `callDepth`: call/create call depth (1024) exceeded
-  `callValueNonExistingAcct`: call with value transfer to a non-existent account

### CODE REVIEW

File: src/GeneralStateTestsFiller/stEWASMTests/createReturnEmptyDataFiller.yml [hunk 1]
```diff
@@ -0,0 +1,142 @@
...
+        (module
+          (import "ethereum" "create" (func $create (param i32 i32 i32 i32) (result i32)))
+          (import "ethereum" "storageStore" (func $storageStore (param i32 i32)))
+          (memory 1)
+          ;; Storage keys
+          (data (i32.const 0)  "\00\00\00\00\00\00\00\00\00\00\00\00\00\00\00\00\00\00\00\00\00\00\00\00\00\00\00\00\00\00\00\00") ;; 0
+          (data (i32.const 32) "\00\00\00\00\00\00\00\00\00\00\00\00\00\00\00\00\00\00\00\00\00\00\00\00\00\00\00\00\00\00\00\01") ;; 1
+          ;; the inner contract to create
+          ;; see WASM source for compiled code, above
+          (data (i32.const 256) "\00\61\73\6d\01\00\00\00\01\09\02\60\02\7f\7f\00\60\00\00\02\13\01\08\65\74\68\65\72\65\75\6d\06\66\69\6e\69\73\68\00\00\03\02\01\01\05\03\01\00\01\07\11\02\06\6d\65\6d\6f\72\79\02\00\04\6d\61\69\6e\00\01\0a\15\01\13\01\02\7f\41\80\04\21\00\41\00\21\01\20\00\20\01\10\00\0b")
```
Comment (axic) [inline src/GeneralStateTestsFiller/stEWASMTests/createReturnEmptyDataFiller.yml]: Could use `extcodecopy` here to remove the need of hardcoding compiled wasm code?

**Comment by axic** [comment]
Looks good apart from that one comment. Haven't validated gas costs myself.

@lrettig @jwasinger fancy a second look?

**Comment by hugo-dc** [comment]
@axic fixed according to your comments

**Comment by axic** [comment]
This looks great imo. I am happy to merge it and do a gas review later.

**Review by axic — APPROVED** [review]

**Pull request merged**